Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang
+
+ADD . /go/src/github.com/salemove/github-review-helper
```

Comment:
> files not used at runtime should not be added to the container.

The only file needed during the runtime is the compiled binary. However, I want to also do the building within the Dockerfile, because I don't want to set up and keep up to date a Go env in our CI and then make sure that it compiles for the right OS and architecture that's going to be used in the container. I could, in theory, use [Rocker](https://github.com/grammarly/rocker) to _build_ and _run_ the container in separate steps, but I don't think it's worth the effort here. This is also the commonly [recommended (this is an official blog)](https://blog.golang.org/docker#TOC_3.) way of doing this in the land of Go.